game/src/js/scenes/TitleScene.js

```diff
@@ -6,6 +6,13 @@ export default class TitleScene extends Phaser.Scene {
 
   constructor() {
     super({key: 'TitleScene'});
+    this.onlineIndicator = this.onlineIndicator.bind(this);
```

Comment:
To make the code more readable, I'd bring this closer to where the bind becomes useful. Lines 14 and 15 should explicitly have `this.onlineIndicator.bind(this);` as parameters.